src/engine/sparqlExpressions/GeoExpression.cpp

```diff
@@ -176,3 +197,21 @@ std::optional<GeoDistanceCall> getGeoDistanceExpressionParameters(
 }
 
 }  // namespace sparqlExpression
+
+// Explicit instantiations for the different geometric relations to avoid linker
+// problems
+using Ptr = sparqlExpression::SparqlExpression::Ptr;
+
+#undef QL_INSTANTIATE_GEO_RELATION_EXPR
```

Comment:
I think sonarcloud wants
```
#ifdef QL_...
#undef QL_...
#endif
``` 
(It doesn't like undef for already  undefined macros)